launcher/service.go

```diff
@@ -0,0 +1,62 @@
+// SPX-License-Identifier: Apache-2.0
+//
+// Copyright (C) 2022 Renesas Electronics Corporation.
+// Copyright (C) 2022 EPAM Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package launcher
+
+import (
+	"github.com/aoscloud/aos_common/aoserrors"
+
+	"github.com/aoscloud/aos_servicemanager/servicemanager"
+)
+
+/***********************************************************************************************************************
+ * Types
+ **********************************************************************************************************************/
+
+type serviceInfo struct {
+	servicemanager.ServiceInfo
+	err error
+}
+
+/***********************************************************************************************************************
+ * Private
+ **********************************************************************************************************************/
+
+func (launcher *Launcher) getCurrentServices(instances []InstanceInfo) {
```

Comment:
the method with get should return some data. In your case, you fill some launcher member. 
You could return data or rename it.

Reply:
renamed to cacheCurrentServices